Makefile

```diff
@@ -155,6 +155,7 @@ endif
 
 resetdb:
 	docker-compose rm --stop --force db
+	docker volume rm warehouse_pgdata
```

Comment:
Thinking about it, I realize `warehouse_pgdata` is actually `${COMPOSE_PROJECT_NAME:-$(basename $(pwd))}_pgdata`. Chance are people didn't change COMPOSE_PROJECT_NAME and their folder is named warehouse but this is just an assumption.

(note that `${COMPOSE_PROJECT_NAME:-$(basename $(pwd))}_pgdata` is the bash syntax, I don't know enough Makefile syntaxt to say what the equivalent would be)

Alternatives include specifying a specific `name` in the volume definition to avoid the compose project name being appended.

I realize this sucks a little bit :( I hadn't foreseen that deriving the volume name would be non-trivial until I saw this diff.

Reply:
Yea, well, need to test this with the `make` first.

Reply:
Here's the make syntax to do it (ok I cheated)
```
$(shell python -c 'import os;print(os.getenv("COMPOSE_PROJECT_NAME", os.getcwd().rsplit("/", 1)[-1]) + "_pgdata")')
```

Reply:
I used standard `make` functions from http://www.gnu.org/software/make/manual/html_node/File-Name-Functions.html

Reply:
See the last commit.